# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: media/webrtc/signaling/src/peerconnection/MediaTransportHandlerIPC.cpp
# Commit: 8196045d38b5
# Full Hash: 8196045d38b50e04313725215c36ba1618c50486
# Author: Byron Campen [:bwc] <docfaraday@gmail.com>
# Date: 2019-09-18 21:49:27
# Regressor Bug: 1569183
# File Overlap Count: 2
# Description:
#   Bug 1569183: Create an ipdl struct called WebrtcProxyConfig, to cut down on the arg count on lots of functions. r=mjf,mayhemer
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D45562
# ==============================================================================

diff -r 6e1eba5c88ad -r 8196045d38b5 media/webrtc/signaling/src/peerconnection/MediaTransportHandlerIPC.cpp
--- a/media/webrtc/signaling/src/peerconnection/MediaTransportHandlerIPC.cpp	Wed Sep 18 14:27:42 2019 +0000
+++ b/media/webrtc/signaling/src/peerconnection/MediaTransportHandlerIPC.cpp	Tue Sep 17 18:19:04 2019 +0000
@@ -169,10 +169,7 @@
       [aProxyConfig = std::move(aProxyConfig), this,
        self = RefPtr<MediaTransportHandlerIPC>(this)](bool /*dummy*/) mutable {
         if (mChild) {
-          mChild->SendSetProxyConfig(dom::TabId(aProxyConfig.GetTabId()),
-                                     aProxyConfig.GetLoadInfoArgs(),
-                                     aProxyConfig.GetAlpn(),
-                                     aProxyConfig.GetProxyPolicy());
+          mChild->SendSetProxyConfig(aProxyConfig.GetConfig());
         }
       },
       [](const nsCString& aError) {});